src/docfx/template/MustacheXrefTagParser.cs

```diff
@@ -41,7 +41,7 @@ private static string ReplaceXrefTag(Match match)
             }
 
             var resolvedTag = partialName == null
-                ? "<a href=\"{{href}}\"> {{uid}} </a>"
+                ? "<a href=\"{{href}}\"> {{name}} {{^name}}{{uid}}{{/name}} </a>"
```

Comment:
instead of changing the template, how about having a `name` property for every xrefspec just like `href`? We have a couple of `GetXrefPropertyValueAsString("name")`, seems to me that `name` is just as special as `href` for xref.

@live1206 , what do you think?

Reply:
@yufeih Cool for me, sounds like a good idea. And can simply used `{{name}}` without a literal fallback.

Reply:
#5668